src/sentry/dynamic_sampling/rules/utils.py

```diff
@@ -208,3 +210,25 @@ def apply_dynamic_factor(base_sample_rate: float, x: float) -> float:
 def get_redis_client_for_ds() -> Any:
     cluster_key = getattr(settings, "SENTRY_DYNAMIC_SAMPLING_RULES_REDIS_CLUSTER", "default")
     return redis.redis_clusters.get(cluster_key)
+
+
+def generate_cache_key_adj_factor(org_id: int) -> str:
+    return f"ds::o:{org_id}:prioritise_projects:adj_factor"
```

Comment:
NIT: this is the adjustment factor for everything ( not specific to prioritise_projects) so I wouldn't call it
`ds::o:123:prioritise_projects:adj_factor` maybe just  `ds::o:123:rate_rebalance_factor` or something similar.

```suggestion
    return f"ds::o:{org_id}:rate_rebalance_factor"
```